Library/Homebrew/cmd/readall.rb

```diff
@@ -13,16 +13,11 @@ module Homebrew
 
   def readall
     if ARGV.include?("--syntax")
-      ruby_files = []
       scan_files = %W[
         #{HOMEBREW_LIBRARY}/*.rb
         #{HOMEBREW_LIBRARY}/Homebrew/**/*.rb
```

Comment:
Since there is no Ruby file at `HOMEBREW_LIBRARY/*.rb`, you could also simplify this to just `HOMEBREW_LIBRARY_PATH/"**/*.rb"`.